test/ViewExtensionLibraryTests/Setup.cs

```diff
@@ -28,7 +28,7 @@ public void SetUp()
         AppDomain.CurrentDomain.AssemblyResolve += assemblyHelper.ResolveAssembly;
     }
 
-    [TearDown]
+    [OneTimeTearDown]
```

Comment:
similarly for teardown

Reply:
I think these attributes on test fixtures were only run once in nunit 2 (so I think this is the correct change) - but it can't hurt to double check the previous before.